crates/holochain_types/src/app/app_manifest/app_manifest_v1.rs

```diff
@@ -68,17 +75,17 @@ pub struct AppSlotDnaManifest {
     /// Note that since this is flattened,
     /// there is no actual "location" key in the manifest.
     #[serde(flatten)]
-    pub(super) location: Option<mr_bundle::Location>,
+    pub location: Option<mr_bundle::Location>,
 
     /// Optional default properties. May be overridden during installation.
```

Comment:
```suggestion
    /// Optional default properties.
    /// Overrides any default properties specified in the DNA file,
    /// and may also be overridden during installation.
    /// A set of properties completely overrides previously specified default properties,
    /// rather than being interpolated into them.
```